src/automyte/utils/config.py

```diff
@@ -0,0 +1,29 @@
+import typing as t
+
+
+from dataclasses import Field, fields, is_dataclass
+
+
+def get_fields_to_process(dataclass, field_name: str) -> t.Sequence[Field]:
```

Comment:
I guess this function could be the part of some `ConfigMetadataParser` class in `src/automyte/config/builders/metadata_parser.py`